### PR TITLE
[MAT-1759] Using VSAC API Key instead of username/password

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: [MAT-1759](https://jira.cms.gov/browse/MAT-1759)
- [ ] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Review by serhii-ilin — APPROVED** [review]

**Review by danmee10 — APPROVED** [review]

File: test/unit/vsac/vsac_api_auth_test.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class VSACAPIAuthTest < Minitest::Test
 
   def test_valid_username_and_password_provided_does_not_raise
     VCR.use_cassette("vsac_auth_good_credentials") do
```
Comment (adongare) [inline test/unit/vsac/vsac_api_auth_test.rb:9]: when you change the credentials from user/password to API key, doesn't this cassette recordings change? similarly for other cassettes

Comment (jkotanchik-SB) [reply test/unit/vsac/vsac_api_auth_test.rb:9]: It does, I missed that.

**Review by adongare — APPROVED** [review]
